lbrynet/reflector/util.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from twisted.internet import reactor, defer
+from lbrynet.reflector import BlobClientFactory, ClientFactory
+
+log = logging.getLogger(__name__)
+
+
+class ReflectorAvailabilityHelper(object):
```

Comment:
Having these methods namespaced inside the class doesn't seem necessary.  Having them be stand-along functions within a module seems cleaner.